### DISCUSSION TITLE
Koin versioning is confusing

### DISCUSSION THREAD

**Discussion by zt64** [InsertKoinIO/koin#1947] [General]
Hi, Koin is a great library but the versioning just seems to be so random and it's not very clear what the latest version is. Intellij and Renovate bot also get confused

As of writing koin-compose has a `4.0.0-RC1` version, released just a few weeks ago but there's also a `1.2.0-Beta5` version released a few days before that. Same with koin-compose-viewmodel

It's just really confusing and should be corrected

**Comment by arnaudgiuliani** [comment]
Yes, this is why it was better to bring all things back to one main version for Koin.  4.x branch is re-aligning all sub projects

**Reply by zt64** [reply]
I see, so 4.x is how it will be going forward?

**Reply by arnaudgiuliani** [reply]
current publishing of RC versions. Final version should be out this month
